content/en/docs/contribute/style/style-guide.md

```diff
@@ -42,12 +42,9 @@ The English-language documentation uses U.S. English spelling and grammar.
 
 ## Documentation formatting standards
 
-### Use camel case for API objects
+### When to use camel case for API objects
 
-When you refer to an API object, use the same uppercase and lowercase letters
-that are used in the actual object name. Typically, the names of API
-objects use
-[camel case](https://en.wikipedia.org/wiki/Camel_case).
+When you are specifically referring to interacting with an API object, use [camel case](https://en.wikipedia.org/wiki/Camel_case). When you are generally discussing an API object, use sentence capitalization.
```

Comment:
Given that many K8s contributors speak English as a second language, we can't take for granted that the meaning of sentence capitalization is obvious. Can you recommend a resource we can link to for more information?

```suggestion
When you refer specifically to interacting with an API object, use [camel case](https://en.wikipedia.org/wiki/Camel_case). When you are generally discussing an API object, use [sentence capitalization]().
```

Reply:
Will the community oust me for referring to a Microsoft document? ;) 

https://docs.microsoft.com/en-us/style-guide/text-formatting/using-type/use-sentence-style-capitalization

Reply:
Why not shorten this to: "When you refer to an API object," ?

Also, the Kubernetes community's documentation has embraced an ambiguous definition of "camel case." The objects in question are capitalized in the first letter and in any subsequent first letters of the closed compound (PodList, e.g.). This is sometimes referred to as "camel case", but when referred to as "Pascal case" is unambiguous. I believe we should standardize on this and describe API objects as being set in Pascal case.

It is at least as accurate, and a lot less confusing.